### PR TITLE
Added more questions on strings

### PR DESCRIPTION
**Pull request by hanzala-sohrab**
Added some more interview questions based on strings. Some questions were taken from the list which can be found [here](https://docs.google.com/document/d/1SM92efk8oDl8nyVw8NHPnbGexTS9W-1gmTEYfEurLWQ/edit).

### CODE REVIEW

**Comment by CubixPro** [comment]
Hi thank you for your interest! Can you add the solutions to some of the links that you posted. Also it would be great if instead of creating a seperate credits heading you would add the link under the links/articles tab.

**Review by TheSYNcoder — CHANGES_REQUESTED** [review]
Please don't try to add new sections in the main README.md, you can add them continuing in a list 
no need of mentioning credits, please keep each point concise.

**Comment by hanzala-sohrab** [comment]
Sure.
I will make the necessary changes.

**Comment by hanzala-sohrab** [comment]
Okay, sure

**Comment by CubixPro** [comment]
This looks great! @TheSYNcoder can you take a look and then we can merge.

File: Topic/BST.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 * [Max level sum](https://leetcode.com/problems/maximum-level-sum-of-a-binary-tree/submissions/), [Solution](Solution/BST1.cpp)
 * [Linked list to BST](https://leetcode.com/problems/convert-sorted-list-to-binary-search-tree/) , [Solution](Solution/BST2.cpp)
 * [Maximum width of Binary Tree ( Bamboo case to be considered ) ](https://leetcode.com/problems/maximum-width-of-binary-tree/) , [Solution](Solution/BST3.cpp)
-
+* [Find K-th smallest and K-th largest element in BST](https://practice.geeksforgeeks.org/problems/find-k-th-smallest-element-in-bst/1), [Solution](https://www.geeksforgeeks.org/find-k-th-smallest-element-in-bst-order-statistics-in-bst/)
+* [K’th Largest element in BST using constant extra space](https://www.geeksforgeeks.org/kth-largest-element-bst-using-constant-extra-space/)
```
Comment (CubixPro) [inline Topic/BST.md]: Solution?

Comment (hanzala-sohrab) [reply Topic/BST.md]: Added solution

**Comment by CubixPro** [comment]
Thank you for your contribution!

**Pull request merged**